app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person_info in people:
```

Comment:
I see what you did here - you actually removed one loop to have less iterations. Good job on that!
But now the code below is a bit repetitive: you have two if blocks that do pretty much the same with only difference that first if uses 'wife' and second uses 'husband'. How would you make it to follow DRY principle?

Also - previous version of code was fine with exception that you could use a list comprehension instead of a for loop.